src/gui/statusbar.h

```diff
@@ -58,8 +58,7 @@ public slots:
 private slots:
     void refresh();
     void updateAltSpeedsBtn(bool alternative);
-    void capDownloadSpeed();
-    void capUploadSpeed();
+    void capSpeed();
```

Comment:
Looks like this is `const` now: `void capSpeed() const;`

Reply:
Same as https://github.com/qbittorrent/qBittorrent/pull/13885#discussion_r540067520
>Can't, parent argument for TorrentOptionsDialog would have to be const and then i can't init QDialog with it...